### ISSUE TITLE
Let's start beginner friendly data structure and algorithms. 

### ISSUE THREAD

**Issue by pravinkori** [fineanmol/hacktoberfest#1052]
🐞 **Problem**
Every aspiring programmer who wants to become good at programming struggles with the complexity of data structures and algorithms.

🎯 **Goal**
Let's begin the series of DS&A to help and guide beginners to understand the core of every programming language and how it works.

💡 **Possible solutions**
We can contribute the algorithms in layman's terms by following good practices in the main repo under a new folder dedicated to only DS&A.

**Comment by picturepatch07** [comment]
I want to contribute to this issue
